eng/Publishing.props

```diff
@@ -13,6 +13,10 @@
     <!-- Include our "loose" PDBs when publishing symbols. -->
     <FilesToPublishToSymbolServer Include="$(ArtifactsDir)\symbols\**\*.pdb" />
 
+    <!-- Ensure the PDBArtifacts artifact isn't empty, avoiding irrelevant build warnings. -->
```

Comment:
Will Arcade try to publish this to MSDL/SymServ?

Reply:
Nope; that post-build step looks for *.pdb in the downloaded PDBArtifacts folder.